README.md

```diff
@@ -132,14 +132,16 @@ After installing tkn, create a link as kubectl-tkn
   ```shell
 $ ln -s /usr/local/bin/tkn /usr/local/bin/kubectl-tkn
   ```
-
-`kubectl` will find any binary named `kubectl-*` on your `PATH` and consider it as a plugin
-
+Run the following to confirm tkn is available as a plugin:
```

Comment:
I don't thank this is needed isnt it ?

Reply:
It's not technically needed, but I think it makes sense as a way to confirm it's available before running `kubectl-tkn`. Better to know it's set up correctly before running commands with it.